user/darkman/template.py

```diff
@@ -0,0 +1,71 @@
+pkgname = "darkman"
+pkgver = "2.0.1"
+pkgrel = 0
+build_style = "go"
+make_build_args = [
+    f"-ldflags=-X main.Version=v{pkgver}",
+    "./cmd/darkman",
+]
+hostmakedepends = [
+    "scdoc",
+    "go",
+]
+pkgdesc = "🌇 control dark-mode and light-mode transitions"
```

Comment:
No emojis in `pkgdesc` please

```suggestion
pkgdesc = "Control dark-mode and light-mode transitions"
```